### ISSUE TITLE
cube reroll martel

### ISSUE THREAD

**Issue by crackedsash** [kolton/d2bot-with-kolbot#1260]
Recently started playing with classic, wondering if its possible to reroll martel by adding this line in character config:

Config.Recipes.push([Recipe.Reroll.Rare, "MartelDefer"]);

or if i have to make a recipe specifically for this in cubing.js?

**Comment by DarkHorseDre** [comment]
Recipe.Reroll.Rare is a definition for a recipe afaik.  the recipe is already defined in the code so just the martel is needed - just ensure you spell the itemname correctly and the recipe works in classic.

**Comment by 5noop** [comment]
Yeah it should work, but be careful because the formula lowers the item level so you won't be able to get max ed or even near max ed.

**Comment by crackedsash** [comment]
If i remember correctly diablo drop should be able to roll 180ed the first time, after that ilvl is basicly busted?

**Comment by 5noop** [comment]
It depends on your character level, basically you can get 180ed on martel if you reroll a dia drop with a char lvl 93+, otherwise it will be 165ed max. Personally I'd rather reroll mage plates and grim helm.

**Comment by crackedsash** [comment]
Oh I has no idea char Level matter! What about rolling amulets then?

**Comment by DarkHorseDre** [comment]
I think it's the same for all rares except diadems

**Comment by drogga** [comment]
Only Diadems

**Comment by DarkHorseDre** [comment]
You right @drogga - been a while since I read this shiz
https://www.diabloii.net/forums/threads/whats-the-recipe-to-reroll-tiaras.480471/
https://forums.tomshardware.com/threads/diadem-vs-tiara-vs-xxx-embue-re-roll-level.151620/
